nodebuilder/store.go

```diff
@@ -150,6 +154,66 @@ type fsStore struct {
 	dirLock *flock.Flock // protects directory
 }
 
+// Finds a single running node and return its store path.
+// Node Type Order: Bridge, Full, Light
+// Network Order: Mainnet, Mocha, Arabica, Private
+func DiscoverOpened() (string, error) {
```

Comment:
array of paths preferred?

Reply:
It's fine to return just the first one. Will extend it multiple once there is a need for that.

Reply:
Mind writing a unit test for it?

Reply:
yup